docs/how-tos/rotating-secrets.md

```diff
@@ -0,0 +1,102 @@
+* **Is it in use?**
+  * In this case, the [Stripe API Keys page in the dashboard](https://dashboard.stripe.com/test/apikeys) has helpful information including when a given valid key was last used. Our example key was last used earlier today, so it's definitely still in use!
+* **What do we use it for?**
+  * Since this is an unrestricted Stripe API key (test mode), it is used to make read/write API requests to any Stripe API in test mode in our staging environment. Without [searching for this key value explicitly in various places](#places-to-check-where-a-secret-might-be-used), one can only guess at this point that it's likely used by our stage apps to make Stripe API calls, and maybe by our CI end-to-end tests to do the same.
+* **What _can_ it be used for?**
+  * In terms of severity, the fact that this key is unrestricted is very bad, but the fact that it's only valid for our staging environment (and in Stripe test mode), which uses fake data and money, greatly reduces the risk and urgency of this leak. 
+    * By contrast, if this same key was for production (live mode), an attacker could [make charges or issue refunds](https://stripe.com/docs/keys#safe-keys) among other things.
+  * **Can someone with this secret access sensitive user data?**
+    * No. All data is test data.
+  * **Can someone with this secret modify critical user or application state?**
+    * They could not modify real users' state. They could modify our stage application state and/or cause us to get rate limited by Stripe, where subsequent API requests are denied.
+* **Where was it exposed?**
+  * The GitHub secret scanning alert indicates where it found the secret referenced. It scans the code but also other forms of user-generated content, like PR comments and issues. It's worth noting that depending on how old the alert is, a reference may have since been removed or moved.
+  * In our case, this was leaked in a user comment within a GitHub issue.
+
+
+## Fix the leak
+
+The most important thing to do -- particularly if the secret is valid -- is to remove the exposed secret from wherever it was found.
+
+In this example, the secret was leaked in a GitHub issue comment. The comment can be edited to redact the reference, but it will still be retained in the comment's revision history. An [admin for the repo](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization) can remove the revision which introduced the leak to completely remove any public reference to it.
+
+## Rotate the secret (if valid)
+
+Now that we've stopped the bleeding, it's time to rotate the secret if it was determined to be valid and in use. 
+
+The steps here are to:
+1. create a new secret,
+2. replace the old secret where it's used with the new secret, and
+3. invalidate the old secret
```

Comment:
Nit: not sure of grammar here, but unordered bullets are capitalized and numbers here are not. Also, semicolon instead of comma, after 'secret' in (2).